frontend/src/v5/ui/themes/theme.ts

```diff
@@ -1029,6 +1029,7 @@ export const theme = createTheme({
 					label: {
 						transform: 'none',
 						position: 'unset',
+						zIndex: 0,
```

Comment:
What is the reason for this?

Reply:
Would it work if you were to increase the z-index of the `IssuePropertiesContainer` instead? Because I think the theme should not change for just one edge case

Reply:
Yeah, that's what I've done